editor/src/components/canvas/canvas-strategies/strategies/fragment-like-helpers.ts

```diff
@@ -171,11 +171,20 @@ export const AllFragmentLikeNonDomElementTypes = ['fragment', 'conditional'] as
 export const AllFragmentLikeTypes = [...AllFragmentLikeNonDomElementTypes, 'sizeless-div'] as const
 export type FragmentLikeType = typeof AllFragmentLikeTypes[number] // <- this gives us the union type of the Array's entries
 
+export interface GetElementFragmentLikeTypeOptions {
+  sizelessDivsConsideredFragmentLike: boolean
```

Comment:
We've been generally trying to avoid boolean fields, instead using unions to make them clearer.
```suggestion
  howToTreatSizelessDivs: 'like-fragments' | 'no-special-handling'
```
Example above likely terribly named because naming things is hard.

Reply:
This is an object for passing options through the chain of function calls, so it will appear like: `someFn({ sizelessDivsConsideredFragmentLike: true })`, which achieves the same goal as the string unions